src/wait-for-dom-change.js

```diff
@@ -10,9 +10,21 @@ function waitForDomChange({
     characterData: true,
   },
 } = {}) {
+  if (typeof container === 'undefined') {
+    if (typeof window === 'undefined') {
+      throw new Error('Could not find default container')
+    } else {
+      container = window.document
+    }
+  }
   return new Promise((resolve, reject) => {
     const timer = setTimeout(onTimeout, timeout)
-    const observer = new window.MutationObserver(onMutation)
+    const MutationObserverConstructor =
```

Comment:
I left this code because I realised this can be useful if someone loads custom MutationObserver after `'@sheerun/mutationobserver-shim'` is already loaded